presto-main-base/src/main/java/com/facebook/presto/server/PluginDiscovery.java

```diff
@@ -38,7 +38,7 @@
 import static java.nio.file.Files.walkFileTree;
 
 // This is a hack for development and does not support nested classes.
-final class PluginDiscovery
+public final class PluginDiscovery
```

Comment:
Same here! No usages of `PluginDiscovery`

Reply:
@auden-woolfson I may be missing something, it seems this is in same package (i.e. `package com.facebook.presto.server;` )as its usages so we won't need to make it public. As the default visibility is package private.